src/lib/network/network_autoyast.rb

```diff
@@ -280,7 +280,7 @@ def assign_udevs_to_devs(udev_rules)
         end
         next if !matching_item
 
-        name_from = matching_item["ifcfg"] || matching_item["dev_name"]
+        name_from = LanItems.GetDeviceName(item)
```

Comment:
fixes an issue when ifcfg is not present but udev rule is defined by user in the profile. The rule cannot be paired properly in such case.